qubesadmin/tools/qvm_create.py

```diff
@@ -43,6 +43,14 @@
     default='AppVM',
     help='specify the class of the new domain (default: %(default)s)')
 
+parser.add_argument('--standalone',
+    action="store_true",
+    help=' shortcut for --class StandaloneVM')
+
+parser.add_argument('--disp'
+    action=qubesadmin.tools.SinglePropertyAction,
```

Comment:
I think this should be just `action="store"` (there is no "disp" property, so `SinglePropertyAction` is not applicable here).
(besides syntax error - missing coma - pointed by travis)
Additionally, in this form the argument isn't optional. For optional, add `nargs=argparse.OPTIONAL`. But see below about using optional arg at all.

Reply:
Sorry, I misunderstood the code there.